promgen/static/js/promgen.vue.js

```diff
@@ -123,43 +128,33 @@ var app = new Vue({
     },
     computed: {
         alertLabelsService: function () {
-            return new Set(this.globalAlerts
-                .filter(x => x.status.state == 'active')
+            return new Set(this.filterActiveAlerts
                 .filter(x => x.labels.service)
                 .map(x => x.labels.service)
-                .sort()
             );
         },
         alertLabelsProject: function () {
-            return new Set(this.globalAlerts
-                .filter(x => x.status.state == 'active')
+            return new Set(this.filterActiveAlerts
                 .filter(x => x.labels.project)
                 .map(x => x.labels.project)
-                .sort()
             );
         },
         alertLabelsRule: function () {
-            return new Set(this.globalAlerts
-                .filter(x => x.status.state == 'active')
+            return new Set(this.filterActiveAlerts
                 .filter(x => x.labels.alertname)
                 .map(x => x.labels.alertname)
-                .sort()
             );
         },
         silenceLabelsService: function () {
-            return new Set(this.globalSilences
-                .filter(x => x.status.state == 'active')
+            return new Set(this.filterActiveSilences
                 .filter(x => x.labels.service)
                 .map(x => x.labels.service)
-                .sort()
             );
         },
         silenceLabelsProject: function () {
-            return new Set(this.globalSilences
-                .filter(x => x.status.state == 'active')
+            return new Set(this.filterActiveSilences
```

Comment:
The behavior here has changed slightly. I saw that `filterActiveSilences` was changed from `state == 'active'` to `state != 'expired'`. I think we want the same behavior here?